piksi_tools/serial_link.py

```diff
@@ -73,68 +74,63 @@ def base_cl_options(override_arg_parse=None, add_help=True):
         action="store_true",
         help="Enable Hardware Flow Control (RTS/CTS).")
     parser.add_argument(
-        "-v",
-        "--verbose",
+        "-t",
+        "--tcp",
         action="store_true",
-        help="print extra debugging information.")
+        default=False,
+        help="Use a TCP connection instead of a local serial port. \
+                      If TCP is selected, the port is interpreted as host:port"
+    )
     parser.add_argument(
```

Comment:
I think we can safely remove the --ftdi option.  It isn't in use by anyone to my knowledge.  Doesn't need to be done as part of this effort but I thought I'd mention it.